src/Examples/UnitModels/Operations/heat_exchanger_NTU_testing.ipynb

```diff
@@ -0,0 +1,513 @@
+   "metadata": {},
+   "source": [
+    "## Fixing input specifications\n",
+    "For this problem, we will fix the inlet conditions and run two different options for unit specifications:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The DOF is 3\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Hot fluid\n",
+    "m.fs.heat_exchanger.hot_side_inlet.flow_mol[0].fix(60.54879) # mol/s\n",
+    "m.fs.heat_exchanger.hot_side_inlet.temperature[0].fix(392.23) # K\n",
+    "m.fs.heat_exchanger.hot_side_inlet.pressure[0].fix(202650) # Pa\n",
+    "m.fs.heat_exchanger.hot_side_inlet.mole_frac_comp[0, \"CO2\"].fix(0.0158) # dimensionless\n",
+    "m.fs.heat_exchanger.hot_side_inlet.mole_frac_comp[0, \"H2O\"].fix(0.8747) # dimensionless\n",
+    "m.fs.heat_exchanger.hot_side_inlet.mole_frac_comp[0, \"MEA\"].fix(0.1095) # dimensionless\n",
+    "\n",
+    "# Cold fluid\n",
+    "m.fs.heat_exchanger.cold_side_inlet.flow_mol[0].fix(63.01910) # mol/s\n",
+    "m.fs.heat_exchanger.cold_side_inlet.temperature[0].fix(326.36) # K\n",
+    "m.fs.heat_exchanger.cold_side_inlet.pressure[0].fix(202650) # Pa\n",
+    "m.fs.heat_exchanger.cold_side_inlet.mole_frac_comp[0, \"CO2\"].fix(0.0414) # dimensionless\n",
+    "m.fs.heat_exchanger.cold_side_inlet.mole_frac_comp[0, \"H2O\"].fix(0.8509) # dimensionless\n",
+    "m.fs.heat_exchanger.cold_side_inlet.mole_frac_comp[0, \"MEA\"].fix(0.1077) # dimensionless\n",
+    "\n",
+    "DOF_initial = degrees_of_freedom(m)\n",
+    "print(\"The DOF is {0}\".format(DOF_initial))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Option 1: Fix HTC and dimensions of each domain\n",
+    "Below, we fix the heat exchanger area, effectiveness factor and heat transfer coefficient, which yields a fully defined problem with zero degrees of freedom that may be initialized and solved:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The DOF is 0\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Unit design variables\n",
+    "m.fs.heat_exchanger.area.fix(100) # m2\n",
+    "m.fs.heat_exchanger.heat_transfer_coefficient.fix(200) # W/m2/K\n",
+    "m.fs.heat_exchanger.effectiveness.fix(0.7) # dimensionless\n",
+    "\n",
+    "m.fs.heat_exchanger.hot_side.deltaP.fix(-2000) # Pa\n",
+    "m.fs.heat_exchanger.cold_side.deltaP.fix(-2000) # Pa\n",
+    "\n",
+    "# Call the degrees_of_freedom function, get final DOF\n",
+    "DOF_final = degrees_of_freedom(m)\n",
+    "print(\"The DOF is {0}\".format(DOF_final))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "tags": [
+     "testing"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "assert DOF_final == 0"
+   ]
+  },
```

Comment:
There should be a (brief) text box here to say that we are going to solve the model.